cmd/pctl/upgrade.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+	"github.com/weaveworks/pctl/pkg/catalog"
+	"github.com/weaveworks/pctl/pkg/git"
+	"github.com/weaveworks/pctl/pkg/runner"
+	upgr "github.com/weaveworks/pctl/pkg/upgrade"
+	"github.com/weaveworks/pctl/pkg/upgrade/repo"
+)
+
+func upgradeCmd() *cli.Command {
+	return &cli.Command{
+		Name:      "upgrade",
+		Usage:     "upgrade profile installation",
+		UsageText: "To upgrade an installation: pctl upgrade pctl-profile-installation-path/ v0.1.1 ",
+		Flags: append(createPRFlags, &cli.StringFlag{
+			Name:  "git-repository",
+			Value: "",
+			Usage: "The namespace and name of the GitRepository object governing the flux repo.",
+		}),
+		Action: func(c *cli.Context) error {
+			// Run upgradeation main
```

Comment:
wft is `upgradeation`?

Reply:
loool. I did a `cp install.go upgrade.go` and `:%s/install/upgrade` :see_no_evil:

Reply:
fixed (rm'd)

Reply:
rm'd :+1: